lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -107,8 +107,10 @@ def adapter
 
       def within(distance, options = {})
         options[:within] = distance
-        #geo_scope(options)
-        with_latlng.where(distance_conditions(options))
+	# Add bounding box to speed up SQL request.
+        bounds = formulate_bounds_from_distance(options, 
+          normalize_point_to_lat_lng(options[:origin]), options[:units] || default_units)
```

Comment:
Line is too long. [89/80]<br>Align the parameters of a method call if they span more than one line.